packages/logger/src/node.ts

```diff
@@ -128,36 +121,34 @@ interface DefaultMeta {
 }
 
 export class WinstonLoggerNode extends WinstonLogger implements LoggerNode {
-  constructor(private readonly opts: LoggerNodeOpts, private readonly transports: winston.transport[]) {
-    const defaultMeta: DefaultMeta = {module: opts?.module || ""};
-    super(
-      winston.createLogger({
-        // Do not set level at the logger level. Always control by Transport, unless for testLogger
-        level: opts.level,
-        defaultMeta,
-        format: getFormat(opts),
-        transports,
-        exitOnError: false,
-        levels: logLevelNum,
-      })
-    );
+  constructor(protected readonly winston: Winston, private readonly opts: LoggerNodeOpts) {
+    super(winston);
+  }
+
+  static fromOpts(opts: LoggerNodeOpts, transports: winston.transport[]): WinstonLoggerNode {
+    return new WinstonLoggerNode(this.createWinstonInstance(opts, transports), opts);
   }
 
   static fromNewTransports(opts: LoggerNodeOpts): WinstonLoggerNode {
-    return new WinstonLoggerNode(opts, getNodeLoggerTransports(opts));
+    return WinstonLoggerNode.fromOpts(opts, getNodeLoggerTransports(opts));
   }
 
-  // Return a new logger instance with different module and log level
-  // but a reference to the same transports, such that there's only one
-  // transport instance per tree of child loggers
```

Comment:
@dapplion This comment should no longer be relevant also it seems inaccurate as we are not returning a new logger instance with different log level.